examples/rfm7xAndBk242xCompatiblity/rfm7x.cpp

```diff
@@ -0,0 +1,906 @@
+//**************************************************************
+//Compiler          : GCC
+//Author            : jnk0le@hotmail.com
+//                    https://github.com/jnk0le
+//License           : MIT
```

Comment:
This is why we need a copy of the MIT License. I quote directly from the license aggreement:
>The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.